src/hooks/usePGPKeyServer.ts

```diff
@@ -31,6 +31,7 @@ const EAS_CONTRACT_ADDRESS = "0xC2679fBD37d54388Ce493F1DB75320D236e1815e"; // Se
 const SCHEMA_REGISTRY_ADDRESS = "0x0a7E2Ff54e76B8E6659aedc9103FB21c038050D0"; // Sepolia 0.26
 
 const graphqlFetch = async (query: string, variables: any) => {
+  //TODO: it should be an environment variable
```

Comment:
🤔 we could also have a mapping with [all the deployments](https://docs.attest.org/docs/quick--start/contracts) and use the selected network here? Wdyt? Or do you think we should commit to only supporting this on Optimism?

Reply:
sidenote: this should probably also use urql. I think this is early code and I try to not use these libs at first not to overcomplicate things. 😅

Reply:
> 🤔 we could also have a mapping with [all the deployments](https://docs.attest.org/docs/quick--start/contracts) and use the selected network here? Wdyt? Or do you think we should commit to only supporting this on Optimism?

Yes i think this would be good.

Reply:
> sidenote: this should probably also use urql. I think this is early code and I try to not use these libs at first not to overcomplicate things. 😅

No problem i will fix it.

Reply:
> 🤔 we could also have a mapping with [all the deployments](https://docs.attest.org/docs/quick--start/contracts) and use the selected network here? Wdyt? Or do you think we should commit to only supporting this on Optimism?

Let me know whats needs to be done

Reply:
let's add another issue to hash this out